servicetalk-encoding-api/src/main/java/io/servicetalk/encoding/api/BufferDecoderGroupBuilder.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.encoding.api;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+import static io.servicetalk.buffer.api.CharSequences.newAsciiString;
+import static java.util.Collections.emptyList;
+
+/**
+ * Builder for {@link BufferDecoderGroup}s.
+ */
+public final class BufferDecoderGroupBuilder {
+    private static final char CONTENT_ENCODING_SEPARATOR = ',';
```

Comment:
Consider adding a space before comma. Helps with readability when users debug and will be consistent with how it's formatted in RFC

Reply:
I prefer to leave it as is. less space on the wire, less white space stripping, negligible readability impact, and consistent with other implementations (e.g. [grpc-java](https://github.com/grpc/grpc-java/blob/master/api/src/main/java/io/grpc/DecompressorRegistry.java#L38))